uxAutoReload replaced with xAutoReload to improve MISRA compliance (#429)
FreeRTOS/FreeRTOS-Kernel: History.txt

Documentation and download available at https://www.FreeRTOS.org/

> 	+ Changed uxAutoReload parameters to timer functions to xAutoReload.  The
> 	  type is now BaseType_t.  This matches the type of pdTRUE and pdFALSE.
> 	  The new function xTimerGetAutoReload() provides the autoreload state as
> 	  a BaseType_.  The legacy uxTimerGetAutoReload was retained with the
> 	  original UBaseType_t return value.

Reviewer: ```suggestion
	+ Changed uxAutoReload parameter in timer functions to xAutoReload.  The
	  type is now BaseType_t.  This matches the type of pdTRUE and pdFALSE.
	  The new function xTimerGetAutoReload() provides the auto-reload state as
	  a BaseType_t.  The legacy function uxTimerGetAutoReload is retained with the
	  original UBaseType_t return value.

```
